### ISSUE TITLE
List Repeater Tooltip broken after upgrade to 6.2.1 on DNN 7.3.4.

### ISSUE THREAD

**Issue by djamell** [DNNCommunity/DNN.Events#4]
I was running 6.1.4 and have a module set to List Mode using a repeater with no tables and a Div Layout with tooltips.

This was working fine until I upgraded to 6.2.1 and now the tooltip appears to be stuffed in a title tag using a table structure.

Before Update (6.1.4)
![tooltipbefore](https://cloud.githubusercontent.com/assets/312711/6921570/59aa2876-d78b-11e4-9ec4-17ab48ba3f69.jpg)
After Update (6.2.1)
![tooltipafter](https://cloud.githubusercontent.com/assets/312711/6921593/70d54238-d78b-11e4-9c40-10676994a822.jpg)

**Comment by EPTamminga** [comment]
Is it possible that the template for the tooltip is overwritten?
Have your cleared you browser cache or tried another browser?

**Comment by djamell** [comment]
I checked the template and it is the same as before.  I have also tried IE, FF, Chrome.  Restarted App Pool in IIS.  All with the same result.

BUT - The tooltip works fine on the Month View.

**Comment by EPTamminga** [comment]
DId you try the defaiult DNN skin and container? And the module on a clean separate page?
AFAIK, there was not really a change in the .js loading.

It might help if the question is posted in Events forum in dnnsoftware.com: 
http://www.dnnsoftware.com/forums/forumid/20/scope/threads

**Comment by djamell** [comment]
Done:  http://www.dnnsoftware.com/forums/forumid/20/threadid/517347/scope/posts

Updated for trying default skin and container (gravity) and single module on one page with no improvement.  No javascript errors observed in any browser.

**Comment by djamell** [comment]
Hello EPT, I'm following up on this bug since posting in the Events Forum.

Roger had the following reply which seemed to indicate the problem.

"OK, I've had a look on my site, it looks like the controls aren't being added correctly to radtooltip manager. For the repeater it should have a control entry that looks like 'ctlEvents_Mod_1072_RptRowBody_5' I think (I'd have to see a working one to be sure), but they all are duplicates of 'dnn_ctr1072_Events_EventRpt'.

I'd suggest you raise this as a bug...

To be honest, I haven't even added the tooltip token to the test on repeater view I just did, so it shouldn't have added the tooltip at all. So it maybe it is adding unnecessary tooltip info that is confusing things. "

The full thread is at http://www.dnnsoftware.com/forums/forumid/20/threadid/517347/scope/posts

**Comment by MaikRichter** [comment]
I have been seeing the same issue where the tooltips did not appear correctly. One workaround that  worked  for me was to modify the EventRpt.ascx file to add AutoTooltipify

ID="toolTipManager" runat="server"  HideEvent="LeaveTargetAndToolTip" Modal="False" EnableShadow="True" CssClass="Eventtooltip" ShowCallout="False" AutoTooltipify="True"  />

However this only works if you just have one event module on a page. Adding a second one broke the tooltip again. Had to disable tooltips on the second module. Also if you allow to switch between repeater and week / month view the tooltips get corrupted on repeater view but work on month view.

**Comment by djamell** [comment]
@EPTamminga - Can you verify that this is a bug that will be fixed?

**Comment by EPTamminga** [comment]
I am on holiday atm (and unable to verify), but there is a pull request that should fix it. If so, that fix will be im the next releases.

**Comment by MaikRichter** [comment]
Great news. Thanks
Maik 


     Ernst Peter Tamminga <notifications@github.com> schrieb am 22:23 Sonntag, 3.Mai 2015:
   

 I am on holiday atm (and unable to verify), but there is a pull request that should fix it. If so, that fix will be im the next releases.—

**Comment by MaikRichter** [comment]
Just another thing I came across working with the module - not sure if you are the one who can fix it too.There are 2 issues with the list index of repeater view

Issue 1Assume you are on page 6 out of 20, watching the list. If you go to detail view on a particual entry and you hit "return" you always get back to the 1st page out of the list, I guess it should return to page 6

Issue 2Select page 6 (or any other) and restrict the selection by picking one category from the list of available ones. Basically the new view is stating you don't have any entries, while you can see that you got 3 pages for this category ( as an example). If you pick the 1st one - or any other - you will get a mixed view from different categories. 

I could provide more details if you want.
Happy holydays...
Maik

 


     Ernst Peter Tamminga <notifications@github.com> schrieb am 22:23 Sonntag, 3.Mai 2015:
   

 I am on holiday atm (and unable to verify), but there is a pull request that should fix it. If so, that fix will be im the next releases.—

**Comment by EPTamminga** [comment]
@MaikRichter: Thanks for the issues, they seem valid/real issues, although not fatal.
It would make life a bit easier if you could make 2 new separate issues here on GitHub OR 2 separate issues on dnnevents.codeplex.com/issues. 
I am still a bit in the transition from CodePlex to GitHub, but expect in the end that CodePlex will only be used for release distribution, which is a bit better than here at GitHub.

**Comment by djamell** [comment]
For the original problem I reported here:

After some testing it seems that this problem was introduced in version 6.3.1. Possible related to a tooltip fix that was implemted?

**Comment by djamell** [comment]
Again, my original issue (this one) was hijacked and the original subject, problem and analysis are on point including my last post above on May 25th.  I also reported this same issue on CodePlex.

Is it necessary that I recreate the issue again in both places?

**Comment by EPTamminga** [comment]
No need to do a split. If someone picks up this issue, I assume the complete comment stream is read and analyzed.

**Comment by relaxabyte** [comment]
I experienced the same thing with 6.2.1 in the repeater list with tooltips only appearing as plain text in the title as djamell originally reported. I see the same repeating control entry that Roger mentioned in the other forum. EPTamminga mentioned a potential fix in the next version back in May 2015. Version 6.2.2 was released in mid-January 2016. After upgrading to 6.2.2, I still see the same problem where the tooltips work fine for the other layouts, but not in the repeater layout.
